docs/book/component-guide/orchestrators/databricks.md

```diff
@@ -0,0 +1,166 @@
+---
+description: Orchestrating your pipelines to run on Databricks.
+---
+
+# Databricks Orchestrator
+
+[Databricks](https://www.databricks.com/) is a unified data analytics platform that combines the best of data warehouses and data lakes to offer an integrated solution for big data processing and machine learning. It provides a collaborative environment for data scientists, data engineers, and business analysts to work together on data projects. Databricks is built on top of Apache Spark, offering optimized performance and scalability for big data workloads.
```

Comment:
I think Spark is nowadays just one component of databricks and I think as the orchestrator has nothing to do with spark, there's no need to mention it